### DISCUSSION TITLE
DDP vs DP on 2 GPUs

### DISCUSSION THREAD

**Discussion by edgarcin95** [asteroid-team/asteroid#610] [Q&A] [answered]
Hi, I have access to a machine with 2 GTX 1080Ti GPUs and I want to train models like DCUNET and SudoRMRF, which PytorchLighting accelerator should I use? DP or DDP?

**Comment by mpariente** [comment, marked as answer]
If they are on the same machine, you can use DP.
See [here](https://pytorch-lightning.readthedocs.io/en/stable/advanced/multi_gpu.html)
